### DISCUSSION TITLE
Latest Update is Hanging Trans on Quit

### DISCUSSION THREAD

**Discussion by iFillip** [transmission/transmission#4787] [General]
Just updated to 4.0.0. When  I select to quit the app becomes unresponsive get the spinning pizza wheel that just keeps spinning. I always have to force quit.
Mac mini late 2012 macOS 10.15.7 Catalina.
How can I go back to the earlier version?
Thank you. Standing by.

**Comment by aliriddle** [comment]
Im finding my system also hang, spinning Bech ball, whilst transferring, and also having to force quit. All since switching to 4.0.0. No such issues on the 3.x.x 

Mac Studio M1 Max

**Reply by iFillip** [reply]
Thank you for posting aliriddle. I hope they fix this. I can't figure out how to download an earlier version, hope someone shares the URL to download the earlier version.

**Comment by ckerr** [comment]
Closing as duplicate of https://github.com/transmission/transmission/issues/4759
